recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - wheel
   run:
     - python
-    - anyio >=3.1.0,<4
+    - anyio >=3.1.0
```

Comment:
What is the reason from removing the upper bound of anyio?
I don't see this has changed, I would leave it. 
https://github.com/jupyter-server/jupyter_server/blob/v1.23.4/setup.cfg#L41

Reply:
You are right. I reverted my change. I'm trying to realize why I changed it...

Reply:
np! thank you